FlowCrypt/Functionality/Services/AttesterApi.swift

```diff
@@ -45,6 +50,8 @@ extension AttesterApi {
             // programming error because should never happen
             throw AppErr.unexpected("Status \(res.status) when looking up pubkey for \(email)")
         }
+        .timeout(Constants.lookupEmailRequestTimeout)
+        .recoverFromTimeOut(result: PubkeySearchResult(email: email, armored: nil))
```

Comment:
done, removed recovering from timeout, so it will throw the timeout error in Promise and client that uses this class will be able to catch that error.